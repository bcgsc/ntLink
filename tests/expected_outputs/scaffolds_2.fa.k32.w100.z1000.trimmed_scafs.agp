ntLink_0	1	21956	1	W	189459	1	21956	+
ntLink_0	21957	22045	2	N	89	scaffold	yes	paired-ends
ntLink_0	22046	35634	3	W	183836	1	13589	-
ntLink_0	35635	36082	4	N	448	scaffold	yes	paired-ends
ntLink_0	36083	53511	5	W	182169	1	17429	-
ntLink_0	53512	54804	6	N	1293	scaffold	yes	paired-ends
ntLink_0	54805	91691	7	W	190964	1	36887	+
