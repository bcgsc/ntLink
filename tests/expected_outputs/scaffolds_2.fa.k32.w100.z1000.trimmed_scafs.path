ntLink_0	189459+ 90N 183836- 449N 182169- 1294N 190964+
