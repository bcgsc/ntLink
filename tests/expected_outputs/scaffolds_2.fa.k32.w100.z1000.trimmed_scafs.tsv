182169	0	17429
183836	0	13589
189459	0	21956
190964	0	36887
